master/buildbot/reporters/gerrit.py

```diff
@@ -300,6 +300,9 @@ def buildComplete(self, key, build):
             result = _handleLegacyResult(result)
             self.sendCodeReviews(build, result)
 
+    def checkConfig(self, *args, **kwargs):
```

Comment:
I have searched a commit that introduced the bug. It was [this commit](https://github.com/buildbot/buildbot/commit/3d2d77095e9a21caab0cbd93298adf103136252e). It looks like the reason that caused the bug was following diff in `BuildbotService` code.
```diff
-    def checkConfig(self, *args, **kwargs):
-        return defer.succeed(True)
+    def checkConfig(self, name=None, *args, **kwargs):
+        if name is not None:
+            self.name = ascii2unicode(name)

```